src/Compilers/Core/Portable/ReferenceManager/CommonReferenceManager.Resolution.cs

```diff
@@ -106,7 +137,12 @@ public int Index
 
             private string GetDebuggerDisplay()
             {
-                return IsSkipped ? "<skipped>" : $"{(_kind == MetadataImageKind.Assembly ? "A" : "M")}[{Index}]: aliases='{string.Join("','", _aliases)}'";
+                return IsSkipped ? "<skipped>" : $"{(_kind == MetadataImageKind.Assembly ? "A" : "M")}[{Index}]:{DisplayAliases(_aliasesOpt, "aliases")}{DisplayAliases(_recursiveAliasesOpt, "recursive-aliases")}";
+            }
+
+            private static string DisplayAliases(ImmutableArray<string> aliasesOpt, string name)
+            {
+                return aliasesOpt.IsDefault ? "" : $" {name} = '{string.Join("','", aliasesOpt)}'";
```

Comment:
I don't think it needs to. We could show aliases=empty instead of aliases='', but that's rather insignificant difference.